permissions/component-fips-bundle-test.yml

```diff
@@ -2,7 +2,7 @@
 name: "lib-fips-bundle-test"
 github: "jenkinsci/lib-fips-bundle-test"
 paths:
-  - "io/jenkins/test/fips/lib-fips-bundle-test"
```

Comment:
weird 
```
Caused by: org.apache.maven.wagon.authorization.AuthorizationException: authorization failed for https://repo.jenkins-ci.org/releases/io/jenkins/test/fips/fips-bundle-test/20.vb_e5df22f62ea_/fips-bundle-test-20.vb_e5df22f62ea_.pom, status: 403 Forbidden
```

here https://github.com/jenkinsci/lib-fips-bundle-test/actions/runs/10845721879/job/30097173717


@timja

Reply:
oh maybe I need to learn patience? :)

Reply:
should run quick but the job does take a bit of time: https://github.com/jenkins-infra/repository-permissions-updater/blob/master/Jenkinsfile#L16

Reply:
no worries, Friday arvo I will have a beer ;)

Reply:
I got few but still a [problem](https://github.com/jenkinsci/lib-fips-bundle-test/actions/runs/10848271899/job/30105055607) (not sure I can get more now :) ) 
I will wait tomorrow.

Reply:
@timja 
still not working https://github.com/jenkinsci/lib-fips-bundle-test/actions/runs/10857213543/job/30133288891

Reply:
I hope there is no mistake somewhere as I may have been confused by the naming convention :)
- github is `lib-fips-bundle-test`
- artifactId `fips-bundle-test`
-  file here is `component-fips-bundle-test.yml`

You guys made it hard to me to not make any mistake :)

Reply:
issue created here https://github.com/jenkins-infra/helpdesk/issues/4290